invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json

```diff
@@ -1,41 +1,7 @@
 {
   "name": "invenio-administration-ui",
   "config": {
-    "languages": [
-      "af",
-      "ar",
-      "bg",
-      "ca",
-      "cs",
-      "da",
-      "de",
-      "el",
-      "en",
-      "es",
-      "et",
-      "et_EE",
-      "fa",
-      "fr",
-      "gl",
-      "hr",
-      "hu",
-      "it",
-      "ja",
-      "ka",
-      "lt",
-      "no",
-      "pl",
-      "pt",
-      "ro",
-      "ru",
-      "rw",
-      "sk",
-      "sv",
-      "tr",
-      "uk",
-      "zh_CN",
-      "zh_TW"
-    ]
+    "languages": ["en"]
```

Comment:
what is this configuration for and why do you have reduced it from all to english only?

Reply:
English is the default language and will not be provided by Transifex, so this is generated on extraction.

Reply:
and the others don't have to be generated on extraction because they will be created on the pull step?

Reply:
Yes, exactly. The non-English translations will be pulled from Transifex and compiled during the `compile_catalog` step, which happens as part of the publish pipeline. Only English is extracted locally because it's the source.